extensions/CheckUser/CheckUser.php

```diff
@@ -67,9 +67,6 @@
 # Take over autoblocking
 $wgHooks['PerformRetroactiveAutoblock'][] = 'CheckUserHooks::doRetroactiveAutoblock';
 
-# Register tables that need to be updated when a user is renamed
-$wgHooks['UserRename::Local'][] = 'CheckUserHooks::onUserRenameLocal';
-
 $wgResourceModules['ext.checkUser'] = array(
 	'scripts'       => 'checkuser.js',
 	'dependencies' 	=> array( 'mediawiki.util' ), // IP stuff
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app